meta-lmp-support/recipes-core/base-files/base-files/rpi/fstab

```diff
@@ -4,11 +4,5 @@ devpts               /dev/pts             devpts     mode=0620,gid=5       0  0
 tmpfs                /run                 tmpfs      mode=0755,nodev,nosuid,strictatime 0  0
 tmpfs                /var/volatile        tmpfs      defaults              0  0
 
-# boot partition
+# Use LABELs to locate partitions
 LABEL=boot           /mnt/boot            vfat       noatime,sync          0  0
-LABEL=config         /mnt/config          auto       defaults              0  0
-LABEL=flags          /mnt/flags           auto       defaults              0  0
-LABEL=cache          /mnt/cache           auto       defaults              0  0
-LABEL=userdata       /userdata            auto       defaults              0  0
```

Comment:
What's the fate of userdata? Will it come from somewhere else, is there a risk it'll be cleared in update?

Reply:
userdata is created in /var/rootdirs/userdata by the recipe in meta-mbed-edge. This area is not touched by the ostree update.

Reply:
OSTree also maintains `/etc`, it will do some kind of fancy 3-way merge with stuff in there.